language-design.md

```diff
@@ -74,7 +79,9 @@ the commenters?
 
 **Response**
 
+The essence of the conflict is between non-programmers who recognizing the difficulty of most programming languages and professional programmers who arguing that if the code is difficult it is for a good reason. While there is truth behind the professional programmer argument, most of the comments demonstrated some degree of hindsight bias. Several of the professional programmers claimed syntax was not hard to pick up, or it was minor in the grand scheme of programming; however, much like how very few native English speakers recognize the difficulty in learning our language because we use it everyday, programmers can just as easily forget how difficult some programming languages can be to learn because they use it everyday. 
```

Comment:
I think you've captured the essence of the conflict well. I have to say, though, I disagree with your comparison of the difficulty of learning programming languages with that of learning a natural language. Speaking as someone who's been learning Japanese since 8th grade, compared to learning a new natural language, learning a new programming language _is_ easy, especially if you've learned a programming language before. I think even learning a first programming language is only especially difficult because you have to learn how things like control flow, loops, and recursion work, and if you understand those for one programming language you have a pretty good idea about them for just about all programming languages. I admit, though, that Python was the first language I used at all extensively, and maybe I got off easy having a first PL that does both functional and object-oriented programming without any complaint, so when I saw (e.g.) Racket for the first time it wasn't completely alien.